base/applications/mspaint/winproc.h

```diff
@@ -29,6 +29,7 @@ class CMainWindow : public CWindowImpl<CMainWindow>
         MESSAGE_HANDLER(WM_MOUSEWHEEL, OnMouseWheel)
     END_MSG_MAP()
 
+protected:
```

Comment:
You decided to limit public visibility here. Generally I am neutral to that.
But why not use **private** then instead of **protected**?
Does anyone inherit from CMainWindow curently? I don't see any further inheritance atm. Or is anyone expected to do so in the future?

Reply:
`s/protected:/private:/`
Fixed.